src/main/js/main_html5.js

```diff
@@ -1016,6 +1017,15 @@ require("../../../html5-common/js/utils/environment.js");
      * @param {object} event The event from the video
      */
     var raiseTimeUpdate = _.bind(function(event) {
+      // [PBW-6200] - MS Edge will not fire 'playing' or 'canplaythrough' events after seeking.
+      // If playback progresses and 'waitingEventRaised' hasn't been cleared, we fire a
+      // 'buffered' event in order to notify that the video has stopped buffering
```

Comment:
What happens when you seek again after an initial seek but before the video is able to play before a buffer? Will timeUpdate be triggered in this case and fire a BUFFERED event?

I think if there exists a reliable event to detect when the video starts playback, we should use it if possible. If canplaythrough does not meet this criteria, is there another event we can listen to? If not, timeupdate sounds good.